### ISSUE TITLE
Request for `NSSavePanel::allowedContentTypes`

### ISSUE THREAD

**Issue by Berrysoft** [madsmtm/objc2#657]
`NSSavePanel::allowedFileTypes` is deprecated and removed from the Apple documents.

**Comment by Berrysoft** [comment]
It is hidden in the content table: https://developer.apple.com/documentation/appkit/nssavepanel?language=objc

And I would like to solve the deprecation warnings instead of just marking them as `allow`

**Comment by madsmtm** [comment]
I think I get it now, I was confused at first because I read `allowedFileTypes` and `allowedContentTypes` as the same thing, but your issue is of course that `objc2-app-kit` does not expose the recommended alternative.

I have fixed this in c85f91efd6d0ab2d435174baa4667f126dfa1c55, in the meantime you can do something like:
```rust
use objc2_app_kit::NSSavePanel;
use objc2_uniform_type_identifiers::UTType;

pub fn allowedContentTypes(panel: &NSSavePanel) -> Retained<NSArray<UTType>> {
    unsafe { msg_send_id![panel, allowedContentTypes] }
}
```
